src/components/PopoverWithMeasuredContent.tsx

```diff
@@ -1,57 +1,22 @@
-import PropTypes from 'prop-types';
+import isEqual from 'lodash/isEqual';
 import React, {useMemo, useState} from 'react';
-import {View} from 'react-native';
-import _ from 'underscore';
+import {LayoutChangeEvent, View} from 'react-native';
 import useWindowDimensions from '@hooks/useWindowDimensions';
 import {computeHorizontalShift, computeVerticalShift} from '@styles/getPopoverWithMeasuredContentStyles';
 import useThemeStyles from '@styles/useThemeStyles';
 import CONST from '@src/CONST';
 import Popover from './Popover';
-import {defaultProps as defaultPopoverProps, propTypes as popoverPropTypes} from './Popover/popoverPropTypes';
-import {windowDimensionsPropTypes} from './withWindowDimensions';
+import {PopoverProps} from './Popover/types';
 
-const propTypes = {
-    // All popover props except:
-    // 1) anchorPosition (which is overridden for this component)
-    // 2) windowDimensionsPropTypes - we exclude them because we use useWindowDimensions hook instead
-    ..._.omit(popoverPropTypes, ['anchorPosition', ..._.keys(windowDimensionsPropTypes)]),
-
-    /** The horizontal and vertical anchors points for the popover */
-    anchorPosition: PropTypes.shape({
-        horizontal: PropTypes.number.isRequired,
-        vertical: PropTypes.number.isRequired,
-    }).isRequired,
-
-    /** How the popover should be aligned. The value you passed will is the part of the component that will be aligned to the
-     * anchorPosition. ie: vertical:top means the top of the menu will be positioned in the anchorPosition */
-    anchorAlignment: PropTypes.shape({
-        horizontal: PropTypes.oneOf(_.values(CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL)),
-        vertical: PropTypes.oneOf(_.values(CONST.MODAL.ANCHOR_ORIGIN_VERTICAL)),
-    }),
-
-    /** Static dimensions for the popover.
-     * Note: When passed, it will skip dimensions measuring of the popover, and provided dimensions will be used to calculate the anchor position.
-     */
-    popoverDimensions: PropTypes.shape({
-        height: PropTypes.number,
-        width: PropTypes.number,
-    }),
+type AnchorPosition = {
+    horizontal: number;
+    vertical: number;
 };
 
-const defaultProps = {
-    ...defaultPopoverProps,
-
-    // Default positioning of the popover
-    anchorAlignment: {
-        horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.LEFT,
-        vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
-    },
-    popoverDimensions: {
-        height: 0,
-        width: 0,
-    },
-    withoutOverlay: false,
-};
+type PopoverWithMeasuredContentProps = {
+    /** The horizontal and vertical anchors points for the popover */
+    anchorPosition: AnchorPosition;
+} & Omit<PopoverProps, 'anchorPosition'>;
```

Comment:
NAB: This is not written down anywhere as a guideline but when extending types we are more often using this form
```suggestion
type PopoverWithMeasuredContentProps = Omit<PopoverProps, 'anchorPosition'> & {
    /** The horizontal and vertical anchors points for the popover */
    anchorPosition: AnchorPosition;
};
```